consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/impl/ClaimStatusUpdateServiceTest.java

```diff
@@ -1,40 +1,47 @@
 package uk.gov.moj.sdt.cmc.consumers.client.impl;
 
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
 import uk.gov.moj.sdt.cmc.consumers.request.ClaimStatusUpdateRequest;
-import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+import uk.gov.moj.sdt.cmc.consumers.response.ClaimStatusUpdateResponse;
 
-import static org.mockito.Mockito.mock;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-class ClaimStatusUpdateServiceTest extends AbstractSdtUnitTestBase {
+class ClaimStatusUpdateServiceTest extends CMCConsumersClientTestBase {
 
-    String IDAM_ID_HEADER = "IDAMID";
-
-    String SDT_REQUEST_ID = "SDTREQUESTID";
+    private ClaimStatusUpdateService claimStatusUpdateService;
 
     @Mock
-    private CMCApi cmcApi;
+    ClaimStatusUpdateRequest mockClaimStatusUpdateRequest;
```

Comment:
```suggestion
    private ClaimStatusUpdateRequest mockClaimStatusUpdateRequest;
```

Reply:
Made this field private.  Also made the response field after it private as well.